unlock-app/src/components/content/event/LandingPage.tsx

```diff
@@ -0,0 +1,226 @@
+import { Disclosure } from '@headlessui/react'
+import { Button } from '@unlock-protocol/ui'
+import { BsPlusLg as PlusIcon } from 'react-icons/bs'
+import { Icon } from '@unlock-protocol/ui'
+import Image from 'next/image'
+import Link from 'next/link'
+
+interface AccordeonProps {
+  title: string
```

Comment:
Wrong spelling. Accordian

Reply:
Oups! That's the french spelling! I had no idea it is different in English!